rust/cardano-chain-follower/src/chain_sync_live_chains.rs

```diff
@@ -339,7 +344,7 @@ impl ProtectedLiveChainBlockList {
         }
 
         // Now find points based on an every increasing Slot age.
-        let mut slot_age: u64 = 40;
+        let mut slot_age: Slot = 40.into();
```

Comment:
Probably should be a constant (I know it wasn't a constant before).
```suggestion
        let mut slot_age: Slot = INITIAL_SLOT_PROBE_AGE;
```